docs/reference/search/more-like-this.asciidoc

```diff
@@ -1,7 +1,12 @@
 [[search-more-like-this]]
 == More Like This API
 
+ifdef::asciidoctor[]
+deprecated[1.6.0, "The More Like This API will be removed in 2.0, instead use the <<query-dsl-mlt-query, More Like This Query>>"]
```

Comment:
It is probably worth doing the right number of `:`s here if we're doing an ifdef anyway.

Reply:
:doh: Fixed with 259adbb.

Reply:
Sorry! I think `::` is right for `ifdef`. I meant for `deprecated`!

Reply:
Adding a `:` to `deprecated` actually changes the output here. I included some screencaps below.

As the goal is to mimic the original output, I'm gonna leave off the `:`.

**Asciidoctor: `deprecated[`**
<img width="758" alt="Asciidoctor - After" src="https://user-images.githubusercontent.com/40268737/58040664-f8c09600-7b03-11e9-8ebd-214117a20272.png">

**Asciidoctor: `deprecated:[`**
<img width="755" alt="Screen Shot 2019-05-20 at 3 06 10 PM" src="https://user-images.githubusercontent.com/40268737/58045651-0c262e00-7b11-11e9-8e07-fa93a0490584.png">